src/explore-education-statistics-admin/src/components/editable/EditableSectionBlocks.module.scss

```diff
@@ -0,0 +1,3 @@
+.openSectionBlock {
+  margin-bottom: 20rem !important;
```

Comment:
Did `min-height` not work? Seems like this would add aggressive amounts of margin that doesn't seem right.